doc/api/url.md

```diff
@@ -859,11 +859,22 @@ new URLSearchParams([
 
 Append a new name-value pair to the query string.
 
-#### `urlSearchParams.delete(name)`
+#### `urlSearchParams.delete(name[, value])`
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/47885
+    description: added optional value argument.
+-->
 
 * `name` {string}
+* `value` {string}
+
+If `value` is provided, then delete all name-value pairs with
```

Comment:
The phrasing should be consistent between the two paragraphs.
```suggestion
If `value` is provided, removes all name-value pairs with
```

Reply:
thanks